src/components/FallbackImage.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react'
+import { Image, ImageProps } from 'antd'
+
+interface FallbackImageProps {
+  src: string | undefined
+  fallbackSrc: string | undefined
+  rest: ImageProps
+}
+
+const FallbackImage = ({ src, fallbackSrc, rest }: FallbackImageProps) => {
+  const [imgSrc, setImgSrc] = useState(src)
+
+  useEffect(() => {
+    setImgSrc(src)
+  }, [src])
+
+  return (
+    <Image
+      {...rest}
+      src={imgSrc ? imgSrc : ''}
+      onError={() => {
+        setImgSrc(fallbackSrc)
```

Comment:
**question** antd Image has a `fallback` prop. I've never used it, but is it useful here potentially?

https://ant.design/components/image/

Reply:
Good call, I was looking at Next Image docs for some reason